HW_3_CONTENT/HW_3-3.1.py

```diff
@@ -0,0 +1,12 @@
+hat_list = [1, 2, 3, 4, 5]
```

Comment:
This style prefere! Try to use it at the **first strings** of your program in the next home works
######################
#Author:
#Date:
#Task:
######################